build.gradle.kts

```diff
@@ -44,19 +44,31 @@ task<Jar>("javadocJar") {
     archiveClassifier.set("javadoc")
 }
 
-task<Exec>("testCmdlineExport-library") {
+task<Exec>("testCmdLineExport-library") {
```

Comment:
Do these work (on unix machines)? I think I never tried ...

Reply:
They do run. Not to completion (see https://github.com/LionWeb-io/lionweb-mps/actions/runs/11139777271/job/30957126805?pr=106), but that's another problem.